docs/how_to/chain-expressions.md

```diff
@@ -0,0 +1,68 @@
+# How to Chain Ibis Expressions using the Deferred Expression API (Also known as the Underscore (`_`) API)
```

Comment:
Let's make the title as concise as possible. How about this?

```suggestion
# Chaining Expressions with `_`
```

Reply:
I think I wanted to avoid just using `_` in the title for SEO purposes per #4704: people probably won't know to call it _ when wanting to chain expressions and I don't know how the _ character would affect searches, but that is basically pure speculation on my part.

What do you think? I do like the more concise title, mostly because the sidebar looks really cluttered right now, but also because I'm not a fan of really long titles.  I'm just worried about how people will _find_ this info.

Reply:
I guess the key is "chain Ibis expressions", and the "with _" will tell them "you can do it using this thing".  I also call out the actual name of the object in the text, so I'm going to go ahead and accept this one.

Thanks @cpcloud !

Reply:
Yeah, it could also be `Chaining Expressions with Underscore`.